ckanext/push_errors/tests/fixtures.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+from unittest.mock import MagicMock
+from ckanext.push_errors.plugin import PushErrorsPlugin
+
+
+@pytest.fixture
+def mock_app():
+    """Simulate an application for testing."""
+    app = MagicMock()
+    app.register_error_handler = MagicMock()
+    app.logger = MagicMock()
+    return app
+
+
+@pytest.fixture
+def plugin():
+    """Create an instance of the PushErrorsPlugin."""
+    return PushErrorsPlugin()
```

Comment:
```suggestion
```

None of this fixtures are required
CKAN already have the `app` fixture for a CKAN instance